cleverhans/utils.py

```diff
@@ -111,7 +111,8 @@ def random_targets(gt, nb_classes):
              encoded as one-hot labels.
     """
     # If the ground truth labels are encoded as one-hot, convert to labels.
-    if len(gt.shape) == 2:
+    from keras.utils import np_utils
```

Comment:
I had fixed it independently to len(gt.shape) > 1, which (I believe) is identical. I can make it == 2 though.